### ISSUE TITLE
Bug displaying SVG files in pythonhosted

### ISSUE THREAD

**Issue by Peque** [pypa/pypi-legacy#480]
I just uploaded my documentation to pythonhosted.org:

https://pythonhosted.org/osbrain/

Note, however, how the SVG logo file is not correctly displayed. I think this is a pythonhosted.org but, as I can correctly visualize the generated documentation if I open it locally.

These browsers fail to load the image: Firefox, Chrome, Android stock (4.4.). qutebrowser is able to show the SVG logo. As the major (most used) browsers fail, this leads me again to a possible pythonhosted.org issue.

Note that the SVG logo is correctly uploaded (as you can right-click on the alt-text and open it in a new tab). This will work with every browser:

https://pythonhosted.org/osbrain/_images/osbrain-logo-name.svg

**Comment by Peque** [comment]
Here is the zipped HTML documentation I uploaded (so you can see it works when you open the `index.html` locally): [html.zip](https://github.com/pypa/pypi-legacy/files/361546/html.zip)

**Comment by ewdurbin** [comment]
```$ curl -I https://pythonhosted.org/osbrain/_images/osbrain-logo-name.svg
HTTP/1.1 200 OK
Server: gunicorn/19.6.0
Date: Wed, 13 Jul 2016 14:10:32 GMT
Connection: keep-alive
Content-Type: text/html; charset=UTF-8
Content-Length: 2989```

it looks like the issue is the `Content-Type` header. For now, I don't think our web servers are "aware" of `svg`. I will look into it, but I think the best workaround would be to use a `png` in the interim.

**Comment by ewdurbin** [comment]
it may be enough to place the instances of [`pypi-docs-proxy`](https://github.com/python/pypi-docs-proxy) behind `nginx` on the nodes to get the automagic `Content-Type` headers.

**Comment by Peque** [comment]
@ewdurbin Yeah, a PNG works fine. However I would rather keep the SVG, as nowadays every browser plays well with this type of files and the resulting logo has perfect quality and much smaller size than a PNG (even for a small 300px-width PNG). Appart from that, it is a text file, which is better for version control.

**Comment by ewdurbin** [comment]
@Peque i understand, but I don't forsee us having resources to address this issue imminently. currently i'd suggest using the `png` workaround for now.

**Comment by Peque** [comment]
@ewdurbin Thanks for your fast response. Could you try to quantify "imminently"? (days, weeks, months, reallynobodyknows...) So that I can decide whether it is worth waiting. :innocent: 

Also, wouldn't you mind? If I put a PNG instead, then the link will work well and this issue would be harder to reproduce, I guess.

**Comment by ewdurbin** [comment]
this is likely weeks out. i'm positive it can be reproduced without you leaving documentation without logos.

**Comment by ewdurbin** [comment]
working on this, a few tests show that just moving our existing deployment of pypi-docs-proxy from python 2.7 to python 3.5 will fix it, as the `mimetypes` standard lib module understands the `.svg` prefix.

a little bit of work to do, but this should be out in the next 24-48 hours.
